Calibration/HcalAlCaRecoProducers/src/AlCaIsoTracksProducer.cc

```diff
@@ -149,7 +149,6 @@ AlCaIsoTracksProducer::AlCaIsoTracksProducer(const edm::ParameterSet& iConfig) :
   eIsolation_                         = iConfig.getParameter<double>("IsolationEnergy");
   labelGenTrack_                      = iConfig.getParameter<edm::InputTag>("TrackLabel");
   labelRecVtx_                        = iConfig.getParameter<edm::InputTag>("VertexLabel");
-  labelBS_                            = iConfig.getParameter<edm::InputTag>("BeamSpotLabel");
```

Comment:
This input tag is still present in the configuration fragment:
https://github.com/cms-sw/cmssw/pull/8943/files#diff-437898eff887c85e588dfc2133baff8dR7